Common/ExecutionModel/vtkThreadedImageAlgorithm.cxx

```diff
@@ -333,4 +448,3 @@ void vtkThreadedImageAlgorithm::ThreadedExecute(
   (void)threadId;
   vtkErrorMacro("Subclass should override this method!!!");
 }
-
```

Comment:
Unsure why this line shows being deleted in git.  It is there in the file as well editing the file shows it as well.